src/plugins/dataflow-tool/nodes/factories/control-rete-node-factory.tsx

```diff
@@ -0,0 +1,114 @@
+import Rete, { Node, Socket } from "rete";
+import { NodeData } from "rete/types/core/data";
+import { DataflowReteNodeFactory, kEmptyValueString } from "./dataflow-rete-node-factory";
+import { ValueControl } from "../controls/value-control";
+import { DropdownListControl } from "../controls/dropdown-list-control";
+import { NodeControlTypes, roundNodeValue } from "../../model/utilities/node";
+import { PlotButtonControl } from "../controls/plot-button-control";
+
+export class ControlReteNodeFactory extends DataflowReteNodeFactory {
+  constructor(numSocket: Socket) {
+    super("Control", numSocket);
+  }
+
+  private heldValue: number | null = null;
+  private result = 0;
```

Comment:
`result` is only ever used in `worker()`, and its saved value is never used. It could be a local variable in that function.

Reply:
Yup, I think I had been using it more universally earlier...changed it.